react-renderer/src/Evaluator.ts

```diff
@@ -285,7 +289,7 @@ export const evalExpr = (
   }
 };
 
-const toTF = (v: Value<number>): Value<Tensor> => {
+const differentiableValue = (v: Value<number>): Value<Tensor> => {
```

Comment:
Should the return type be `Value<Variable>`?

Reply:
Yeah, Tensor is fine for typechecking and doesn’t really do anything to the actual values. I’ll change it

Reply:
Turns out that not all values are `Variable`s. So I'll keep this for now.